shardingsphere-distribution/shardingsphere-proxy-distribution/src/main/resources/bin/start.sh

```diff
@@ -38,6 +38,17 @@ JAVA_MEM_OPTS=" -server -Xmx2g -Xms2g -Xmn1g -Xss256k -XX:+DisableExplicitGC -XX
 
 MAIN_CLASS=org.apache.shardingsphere.proxy.Bootstrap
 
+print_usage() {
```

Comment:
What's this for?  I suggest delete it from this pr. If you would like to add help, please create a new issue to do this.

Reply:
got it